src/nexosim/_simulation.py

```diff
@@ -198,13 +198,9 @@ def step_unbounded(self) -> MonotonicTime:
         """Iteratively advances the simulation time until the simulation end, as
         if by calling [Simulation.step][nexosim.Simulation.step] repeatedly.
 
-        For real-time clock simulation will end on calling `halt` only (`halt`
-        is effective on the next event, so bench constructor or user shall
-        ensure periodic events with appropriately small period), for non
-        real-time clock simulation will end on exhausting scheduled events or
-        calling `halt`.
+        The request blocks until all scheduled events are processed or
+        the simulation is halted.
 
-        This method blocks other step* and process* requests until completed.
```

Comment:
Add edited line:

This method blocks other `step*` and `process*` requests until completed.